### DISCUSSION TITLE
Effect of No Shared Spectra

### DISCUSSION THREAD

**Discussion by SamuelLozanoJuarez** [vdemichev/DiaNN#1264] [Q&A] [answered]
Hi,
I am using DIA-NN to test a library of modified peptides that I have created with some _E. coli_ samples. I have read that it is recommended to leave the _‘No shared spectra’_ option checked in all runs. However, I have tried running the DIA-NN search with and without _‘No shared spectra’_ checked and the number of distinct peptidoforms identified is higher when this option is unchecked.

Specifically, the library I have created has ~50000 distinct peptidoforms, if I leave the _‘No shared spectra’_ option checked DIA-NN identifies about 18000 peptidoforms from the library, while if I uncheck the option DIA-NN finds 38000 peptidoforms (these values are obtained from the Modified.Sequence field of the report.pr_matrix). In my case it seems to give better results without checking the _‘No shared spectra’_ option.

Is it possible that I am misinterpreting the data? How does the _‘No shared spectra’_ option affect?

Thank you very much.
Samuel

**Comment by vdemichev** [comment]
Hi Samuel,

Yes, number of IDs is higher with unchecked, this option is reducing IDs by primarily removing false IDs. Some peptidoform-related benchmarks here: https://github.com/vdemichev/DiaNN/blob/master/Presentations/HUPO%202024%20poster.pdf.
So the option is basically essential, there are no known scenarios when it's better to have it off.

Best,
Vadim

**Comment by SamuelLozanoJuarez** [comment]
Thank you for the quick response. Just to be sure, if a search is performed without ticking the _‘No shared spectra’_ option, can peptidoforms that have been identified and pass the FDR be considered as correct? Or is there no guarantee that they are correct.

Thank you very much.
Best,
Samuel

**Reply by vdemichev** [reply, marked as answer]
> can peptidoforms that have been identified and pass the FDR be considered as correct?

No.
